ui/ducks/send/send.test.js

```diff
@@ -656,11 +657,10 @@ describe('Send Slice', () => {
         const draftTransaction = getTestUUIDTx(result);
 
         expect(draftTransaction.recipient.error).toStrictEqual(
-          'invalidAddressRecipient',
+          INVALID_RECIPIENT_ADDRESS_NOT_ETH_NETWORK_ERROR,
         );
       });
```

Comment:
This indicates a change in behavior, brought on by the change that @legobeat suggested (which was a net positive change) -- the reason for this is the chainId not being set in the test mock state. If no other e2e tests fail because of this change then its probably fine to just modify the state in the mock to have a chainId set -- if, however, there are failing E2E tests its probably best to keep the original behavior of returning true if the chainId isn't set and then add an issue for tackling the erroneous function logic as @legobeat suggested in his change request.

Reply:
Looks like just a failing lint test, not E2E so its probably safe to just add the chainId into the mock state in the send tests so that the errors thrown match the anticipated result